pages/airdrop/claim.vue

```diff
@@ -303,7 +315,7 @@ export default class AirdropClaimPage extends Vue {
     this.missionLoadingStatus = 'Loading'
     const res: any = await this.$axios
       .post(
```

Comment:
Can we block this action if airdrop end is true?

Reply:
Blocked, also blocked from the server